src/Illuminate/Mail/Mailer.php

```diff
@@ -520,7 +521,9 @@ protected function sendSwiftMessage($message)
         try {
             return $this->swift->send($message, $this->failedRecipients);
         } finally {
-            $this->forceReconnection();
+            if (Config::get('app.is_run_from_queue_daemon', false)) {
```

Comment:
We can't use a facade like this here since the mailer component also can act as a standalone component.